Sources/MongoSwift/Operations/FindOperation.swift

```diff
@@ -188,6 +194,7 @@ public struct FindOptions: Codable {
         self.collation = findOneOptions.collation
         self.comment = findOneOptions.comment
         self.hint = findOneOptions.hint
+        self.`let` = findOneOptions.`let`
```

Comment:
do we need the backticks in either case here, since neither is an isolated `let`?